lib/hawkular/operations/operations_api.rb

```diff
@@ -128,46 +128,113 @@ def invoke_specific_operation(operation_payload, operation_name, &callback)
       invoke_operation_helper(operation_payload, operation_name, &callback)
     end
 
-    # Deploys a war file into WildFly
+    # Deploys an archive file into WildFly
     #
     # @param [Hash] hash Arguments for deployment
     # @option hash [String]  :resource_path canonical path of the WildFly server into which we deploy
     # @option hash [String]  :destination_file_name resulting file name
     # @option hash [String]  :binary_content binary content representing the war file
-    # @option hash [String]  :enabled whether the deployment should be enabled or not
+    # @option hash [String]  :enabled whether the deployment should be enabled immediately, or not (default = true)
+    # @option hash [String]  :force_deploy whether to replace existing content or not (default = false)
+    # @option hash [String]  :server_groups comma-separated list of server groups for the operation (default = ignored)
```

Comment:
Does that mean no server-group  = standalone usage? Or domain usage by only uploading into the domain-controller content repository?